packages/client/hmi-client/src/components/widgets/tera-concept.vue

```diff
@@ -55,7 +55,7 @@ function saveConcepts() {
 		}
 
 		if (!isEmpty(contextConcepts)) {
```

Comment:
Variable using the word `context` shall be updated to.

```suggestion
		if (!isEmpty(modifiersConcepts)) {
```

Reply:
thanks good catch!